browser-log.graphqls

```diff
@@ -0,0 +1,75 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# The list of browser error log
+type BrowserErrorLogBrief {
+    errorLogs: [BasicBrowserErrorLog!]!
+    total: Int!
+}
+
+# Browser error log basic info
+type BasicBrowserErrorLog {
+    uniqueId: String!
+    pagePath: String!
+    time: Long!
+    category: ErrorCategory!
+}
+
+# Represent the conditions used for query BrowserErrorLogBrief
+input BrowserErrorLogQueryCondition {
+    # The value of 0 means all services.
+    serviceId: ID
+    serviceVersionId: ID
+    pagePathId: ID
+    pagePath: String
+    category: ErrorCategory
+    # The time range of browser perf data time
+    queryDuration: Duration
+    paging: Pagination!
+}
+
+# Represent the browser error log
+type BrowserErrorLog {
+    uniqueId: String!
+    service: String!
+    serviceVersion: String!
+    time: Long!
+    pagePath: String!
+    category: ErrorCategory!
+    catalog: String
+    grade: String
+    message: String
+    line: Int
+    col: Int
+    stack: String
+    errorUrl: String
+    firstReportedError: Boolean!
+}
+
+enum ErrorCategory {
+    ALL,
+    AJAX,
+    RESOURCE,
+    VUE,
+    PROMISE,
+    JS,
+    UNKNOWN
+}
+
+extend type Query {
+    queryBasicBrowserErrorLog(condition: BrowserErrorLogQueryCondition): BrowserErrorLogBrief
+    queryBrowserErrorLog(uniqueId: String!): BrowserErrorLog
```

Comment:
Why we need two steps to query logs? This seems strange.

Reply:
>  queryBasicBrowserErrorLog(condition: BrowserErrorLogQueryCondition): BrowserErrorLogBrief

`queryBasicBrowserErrorLog` is used to query log list, reference [trace#queryBasicTraces](https://github.com/apache/skywalking-query-protocol/blob/f38def1d502327856c1cae7ceb233f3c0c8c8e2a/trace.graphqls#L126). I will change into `queryBasicBrowserErrorLogs`

>  queryBrowserErrorLog(uniqueId: String!): BrowserErrorLog

`queryBrowserErrorLog` is used to query log details, reference[trace#queryTrace](https://github.com/apache/skywalking-query-protocol/blob/f38def1d502327856c1cae7ceb233f3c0c8c8e2a/trace.graphqls#L127).


@Fine0830 Do we need that?

Reply:
Yep..

Reply:
Hi @wu-sheng @Fine0830 Based on the current protocol, I'll start writing backend code.

Reply:
I still confuse about, why need two queries? Trace is different, the first query is for segments, the second level is for spans. Could you indicate why we really need two steps?